manifests/qpid_client.pp

```diff
@@ -0,0 +1,45 @@
+# Pulp Master Certs configuration
+class certs::qpid_client (
+
+  $hostname   = $::certs::node_fqdn,
+  $generate   = $::certs::generate,
+  $regenerate = $::certs::regenerate,
+  $deploy     = $::certs::deploy,
+
+  $messaging_ca_cert     = $certs::ca_cert,
+  $messaging_client_cert = $certs::params::messaging_client_cert
+
+  ) inherits pulp::params { # lint:ignore:inherits_across_namespaces
+
+  cert { "${::certs::qpid_client::hostname}-qpid-client-cert":
+    hostname      => $::certs::qpid_client::hostname,
+    common_name   => 'pulp-qpid-client-cert',
```

Comment:
All instances of `$::certs::qpid_client::hostname` can be renamed to `$hostname` because its a local parameter